### PR TITLE
Cathook gdb, Toolbox script, Wait for serverbrowser

### PR DESCRIPTION
**Pull request by SM9CC**
Added cathook gdb, Toolbox script (Thanks gamesneeze), Wait for serverbrowser to allow early injection.

### CODE REVIEW

**Comment by wemanzoz** [comment]
what does this do exactly

**Comment by SM9CC** [comment]
You can use the script to load, build etc on Linux, the cathook gdb which is included works around a bug on Arch Linux, the wait for server browser allows the script to inject as soon as the csgo process becomes available without crashing.

File: Osiris/Hooks.cpp [hunk 1]
```diff
@@ -712,6 +713,12 @@ static int pollEvent(SDL_Event* event) noexcept
 
 Hooks::Hooks() noexcept
 {
+#ifdef _WIN32
+    while (!GetModuleHandleA("serverbrowser.dll")) std::this_thread::sleep_for(std::chrono::milliseconds(50));
```
Comment (mkot2) [inline Osiris/Hooks.cpp:717]: this Hooks::Hooks is only run on linux, put this into the other Hooks::Hooks

Comment (SM9CC) [reply Osiris/Hooks.cpp]: Done thanks!

**Comment by cstles** [comment]
u sure this would work? it uses a dll

**Pull request closed without merging**